Makefile

```diff
@@ -76,8 +76,9 @@ endif
 # we don't tag versions. Override using the same versioning we apply to OperatorHub builds:
 # v{major}.{minor}.{commitcount}-{sha}
 # Note that building against a local commit may result in {major}.{minor} being rendered as
-# `UnknownBranch`. However, the {commitcount} and {sha} should still be accurate.
-SOURCE_GIT_TAG := $(shell export HOME=$(HOME); python3 -m ensurepip >&2; python3 -mpip --no-cache install --user gitpython pyyaml >&2; hack/version2.py)
+# 0.0.x-y if it is an `UnknownBranch`. However, the {commitcount} and {sha}
+# should still be accurate.
+SOURCE_GIT_TAG := $(shell hack/version2.sh)
```

Comment:
Love this. It was super annoying waiting for pip every time the Makefile loaded up.

However, this may have been the only thing ensuring that `pyyaml` was installed when running [generate-saas-template.py](https://github.com/openshift/hive/blob/00b13a6365cc4107aababc93067b2794d92e0339/Makefile#L243). I'm going to look into porting that to bash/yq as well, but in the meantime, can we add the `ensurepip` and `pyyaml` business inside that rule? (At least then it will only run when that rule is invoked.)